utilities/db-migration/migrate/migrate_db.go

```diff
@@ -12,41 +12,56 @@ import (
 	db "github.com/redhat-appstudio/managed-gitops/backend-shared/config/db"
 )
 
-func Main(op_type string) {
+func Main(op_type string, call_type string) bool {
 	addr, password := db.GetAddrAndPassword()
 	port := 5432
 	ctx := context.Background()
 	log := logger.FromContext(ctx)
+	migration_path := ""
+	if call_type == "backend" {
+		migration_path = "file://../utilities/db-migration/migrations/"
+	} else {
+		migration_path = "file://migrations/"
+	}
+	fmt.Printf("%s", migration_path)
 	m, err := migrate.New(
-		"file://migrations/",
+		migration_path,
 		fmt.Sprintf("postgresql://postgres:%s@%s:%v/postgres?sslmode=disable", password, addr, port))
 	if err != nil {
 		log.Error(err, fmt.Sprintf("%v", err))
+		return false
 	}
 	if len(op_type) > 0 {
 		// op_type := os.Args[1]
 		if op_type == "drop_smtable" {
 			dbq, err := db.ConnectToDatabaseWithPort(true, "postgres", port)
 			if err != nil {
 				log.Error(err, fmt.Sprintf("%v", err))
+				return false
```

Comment:
Can replace all of these log+returns, with, for example,  `return fmt.Errorf("unable to connect to DB: %v", err)`